src/main/resources/db/changelog/changes/10-create-shopping-cart-table.yaml

```diff
@@ -0,0 +1,25 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: opopovich
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_shopping_carts_user_id
+                    references: users(id)
+      rollback:
+        - dropTable:
+            tableName: shopping_carts
```

Comment:
```suggestion
            tableName: shopping_carts

```
add empty line wherever you see this red symbol in your PR

Reply:
![image](https://github.com/opopovych/onlineBookStore/assets/49767049/4daee2ae-a1cd-4625-a051-38054ac9f0c0)